src/map/utils/zoneutils.h

```diff
@@ -53,6 +53,7 @@ namespace zoneutils
     CNpcEntity*  GetTrigger(uint16 TargID, uint16 ZoneID);
     CBaseEntity* GetEntity(uint32 ID, uint8 filter = -1);
     CCharEntity* GetCharByName(int8* name);
+    const NPCEntityList_t GetNPCsByName(std::string const& pattern); // returns all npcs matching the given pattern (use % for wildcard), in all zones
```

Comment:
i did not "re-align" the other methods to match this one. Should I?

Reply:
If you're returning a big type, you can save yourself from realigning by returning auto here:
```cpp
auto GetNPCsByName(std::string const& pattern) -> NPCEntityList_t const;
```

You can keep the definition as is, it should play nicely with auto

Reply:
Ah. i was wondering what the purpose of `->` return style was. Thanks.

Reply:
Done